Examples/Detectors/TelescopeDetector/include/ActsExamples/TelescopeDetector/BuildTelescopeDetector.hpp

```diff
@@ -0,0 +1,38 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/TrackingGeometry.hpp"
+#include "Acts/Utilities/BinUtility.hpp"
+#include "ActsExamples/TelescopeDetector/TelescopeDetectorElement.hpp"
+
+#include <vector>
```

Comment:
You should include `#include <memory>` since you use `std::uniqe_ptr`.

Reply:
Fixed.